src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,45 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $item-gap;
+  width: ($item-size * 8) + ($item-gap * 6);
+  margin: 0 auto;
+  padding-top: 50px;
```

Comment:
You should have all paddings 10px, it mentioned in task. So to add more space at top of the calendar, add top margin to container of calendar (body, for example; or create div container; or use mix)

![image](https://user-images.githubusercontent.com/111275759/194113977-278396a2-ece7-4507-af98-da7539b9d8e0.png)